### PR TITLE
Expose setting supplement

### PR DESCRIPTION
**Pull request by chenchao1407**
My bad. Previously in expose setting pull request, I lost 4 commits after rebase. These 4 commits are the lost ones. Probably you can just do a quick review since we have already reviewed these 4 commits.

GearVRf-DCO-1.0-Signed-off-by: Chao Chen

### CODE REVIEW

File: GVRf/Sample/gvrexposeapisample/src/org/gearvrf/gvrexposeapisample/SampleCubeScript.java [hunk 1]
```diff
@@ -47,16 +41,20 @@
     private GVRSceneObject mFrontFace = null;
     private GVRSceneObject mFrontFace2 = null;
     private GVRSceneObject mFrontFace3 = null;
-
+    private static final float SCALE_FACTOR = 2.0f;
     @Override
     public void onInit(GVRContext gvrContext) {
         mGVRContext = gvrContext;
+        
+        Log.d("okokokok","width" + gvrContext.getActivity().getAppSettings().getFramebufferPixelsHigh());
```
Comment (jason2kim) [inline GVRf/Sample/gvrexposeapisample/src/org/gearvrf/gvrexposeapisample/SampleCubeScript.java]: okokokok => SampleCubeScript

File: GVRf/Sample/gvrexposeapisample/src/org/gearvrf/gvrexposeapisample/SampleCubeScript.java [hunk 2]
```diff
@@ -146,7 +144,23 @@ public void onInit(GVRContext gvrContext) {
...
+        GVRSceneObject sphere = new GVRSceneObject(gvrContext,
+                futureSphereMesh, futureCubemapTexture);
+        sphere.getRenderData().setMaterial(cubemapReflectionMaterial);
+        sphere.setName("sphere");
+        scene.addSceneObject(sphere);
+        sphere.getTransform()
+                .setScale(SCALE_FACTOR, SCALE_FACTOR, SCALE_FACTOR);
+        sphere.getTransform().setPosition(0.0f, 0.0f, -CUBE_WIDTH * 0.25f);
         for (GVRSceneObject so : scene.getWholeSceneObjects()) {
             Log.v("", "scene object name : " + so.getName());
```
Comment (jason2kim) [inline GVRf/Sample/gvrexposeapisample/src/org/gearvrf/gvrexposeapisample/SampleCubeScript.java]: "" => SampleCubeScript

**Comment by chenchao1407** [comment]
Thanks Jason. Updated.

**Comment by chenchao1407** [comment]
Any comments now?  @jason2kim  @thomasflynn

**Comment by jason2kim** [comment]
No, it looks good for me.

**Pull request merged**